gossip/gossip.go

```diff
@@ -300,25 +301,23 @@ func (g *Gossip) hasIncoming(addr net.Addr) bool {
 	return g.incoming.hasAddr(addr)
 }
 
-// parseBootstrapAddresses parses the gossip bootstrap addresses
+// initializeBootstrapAddresses parses the gossip bootstrap addresses
```

Comment:
Actually, I think we need to rewrite the comment entirely based on this change. How about:

    // initializeBootstrapAddresses resolves TCP addresses supplied to the constructor
    // or set using setBootstrap(). Non-TCP and resolvable TCP addresses are added to the
    // bootstraps addrSet, except for the local node, which is removed, if listed as a bootstrap host.